packages/utils/src/index.js

```diff
@@ -1,7 +1,32 @@
 // @flow
 import type { RegisteredCache, EmotionCache, SerializedStyles } from './types'
 
-let isBrowser = typeof document !== 'undefined'
+const isBrowser = typeof document !== 'undefined'
+
+const getGlobalInstances = (context: any) => {
+  const globalVersionsKey = '__EMOTION_VERSIONS__'
+  if (!context[globalVersionsKey]) {
+    context[globalVersionsKey] = {}
+  }
+  return context[globalVersionsKey]
+}
+
+type ModuleRegister = { [key: string]: boolean }
+
+const globalContext = isBrowser ? window : global
+
+const instances: ModuleRegister = getGlobalInstances(globalContext)
+
+export function registerEmotionModule(name: string) {
+  if (process.env.NODE_ENV !== 'production') {
```

Comment:
tried using [`babel-plugin-dev-expression`](https://github.com/4Catalyzer/babel-plugin-dev-expression), but it was causing failures when running the test suite.